### PR TITLE
Add "secondary status" to Linodes

### PR DESCRIPTION
**Pull request by johnwcallahan**
## Description

Adds support for states like "disk duplicating" and "cloning" to the Entity Detail header.
Screen Shot 2020-12-14 at 9 41 10 AM

### CODE REVIEW

File: packages/manager/src/features/linodes/LinodeEntityDetail.tsx [hunk 1]
```diff
@@ -39,11 +42,20 @@ import { sendLinodeActionMenuItemEvent } from 'src/utilities/ga';
...
+  getProgressOrDefault,
+  isEventWithSecondaryLinodeStatus,
+  transitionText as _transitionText
+} from './transitions';
 
 type LinodeEntityDetailVariant = 'dashboard' | 'landing' | 'details';
 
 interface LinodeEntityDetailProps {
   variant: LinodeEntityDetailVariant;
+  id: number;
```
Comment (johnwcallahan) [inline packages/manager/src/features/linodes/LinodeEntityDetail.tsx:58]: The withRecentEvent HOC required an ID prop, hence the duplication (the Linode ID also exists under the "linode" prop). 

Probably would have been just as much effort to write a corresponding hook, honestly.

**Review by Jskobos — APPROVED** [review]
I'm pretty sure we discussed this and decided it was kosher, double checking:

Screen Shot 2020-12-14 at 10 30 41 AM

**Review by DevDW — APPROVED** [review]
Looks good 👌 

With the Linodes landing summary view in mobile, I noticed that the statuses and ellipsis feel a bit jumbled:

Screen Shot 2020-12-14 at 12 22 36 PM

Maybe the statuses and ellipsis could be placed on the line below the Linode label instead?

**Comment by johnwcallahan** [comment]
> Maybe the statuses and ellipsis could be placed on the line below the Linode label instead?

Good catch @DevDW, @tiffwong is going to have a look at this separately.

**Pull request merged**